app/src/org/commcare/google/services/analytics/FirebaseAnalyticsUtil.java

```diff
@@ -123,11 +128,6 @@ private static void setUserProperties(FirebaseAnalytics analyticsInstance) {
         }
 
         analyticsInstance.setUserProperty(CCAnalyticsParam.BUILD_NUMBER, String.valueOf(BuildConfig.VERSION_CODE));
-
-        Boolean isPersonalIDDemoUser = ReportingUtils.getIsPersonalIDDemoUser();
-        if (isPersonalIDDemoUser != null) {
-            analyticsInstance.setUserProperty(CCAnalyticsParam.IS_PERSONAL_ID_DEMO_USER, String.valueOf(isPersonalIDDemoUser));
-        }
```

Comment:
why did we decide to remove this from here and add it somewhere else ?

Reply:
>  I'm calling the wrapper function from inside reportEvent() instead of setUserProperties() because I think it makes more sense from a readability standpoint

This is what I am concerned about as it creates inconsistency with how other user properties are handled. Think user properties doesn't generally change properties across events, which is why we have not been setting them from reportEvent methods.